docker/fc34/support-simx.sh

```diff
@@ -18,7 +18,7 @@ From simx.git
 %build
 ./mlnx_infra/config.status.mlnx --target=x86 --prefix=/opt/simx
 make %{?_smp_mflags}
-make %{?_smp_mflags} -C mellanox/
+make %{?_smp_mflags} -C mellanox/ SIMX_PROJECT=mlx5
```

Comment:
This tells SimX to only build the NIC part. I think it makes sense unless the switch part is planned to be used.

Reply:
Yeah, it makes sense for now. A long time ago, I pitched this project to switch team, they even tried it, but decided to stick with VMs because of differences in technical level expertise between development team and verification team.